zerver/tests/test_upload_local.py

```diff
@@ -86,7 +87,35 @@ def test_delete_message_attachment(self) -> None:
 
         response_dict = self.assert_json_success(result)
         path_id = re.sub("/user_uploads/", "", response_dict["uri"])
+
+        assert settings.LOCAL_FILES_DIR is not None
+        file_path = os.path.join(settings.LOCAL_FILES_DIR, path_id)
+        self.assertTrue(os.path.isfile(file_path))
+
         self.assertTrue(delete_message_attachment(path_id))
+        self.assertFalse(os.path.isfile(file_path))
+
+    def test_delete_message_attachments(self) -> None:
+        assert settings.LOCAL_UPLOADS_DIR is not None
+        assert settings.LOCAL_FILES_DIR is not None
+
+        user_profile = self.example_user("hamlet")
+        path_ids = []
+        for n in range(1, 1005):
```

Comment:
How long does this take to run? We could consider making the BATCH_SIZE a global that we can mock for a test instead if it takes a second or something.

Reply:
It takes a couple seconds -- I'll pull out a constant to mock.

Reply:
Actually, we don't need the 1000 magic number, because the two places with that batch size limit aren't actually tested -- they're the `delete_old_unclaimed_attachments` and `delete_realm` management commands.